scripts/archinstall/arch/archinstall.py

```diff
@@ -47,6 +47,10 @@ def run(self, command, capture=False, **kwargs):
         if capture:
             return subprocess.check_output(command, **kwargs).decode('utf-8')
         ret = subprocess.run(command, **kwargs)
+        # TODO : a critical flag to let the caller choose the fate of the
+        # current script execution. If a non critical part of the script
+        # fail we just want a message displayed, not a full fail
+        # Could be done just with try ... except on this error when non critical
```

Comment:
done. added a critial=True by default since we want a realy proper install, failure inside a command is definitly not ok and should lead to fix the configuration before trying to continue